### DISCUSSION TITLE
OpenCTI : Missing Threat Actor Entities feeds

### DISCUSSION THREAD

**Discussion by staphysec** [OpenCTI-Platform/opencti#5299] [General]
Hello, in OpenCTI, I've noticed that all reports/feeds/ connectors  are inserting intrusion-sets, but threat actors are consistently empty. Is there a specific Cyber Threat Intelligence (CTI) reason behind why connectors are adding intrusion-set entities but not including threat actor entities?

**Comment by SunnyHazuki** [comment]
Same here

**Comment by Henning15** [comment]
Hello... just wondering if you got a way to remedy this please?
A link to guide me perhaps..?
